libcontainer/nsenter/nsexec.c

```diff
@@ -37,27 +37,27 @@
 
 /* mount_setattr() */
 #ifndef MOUNT_ATTR_IDMAP
-#define MOUNT_ATTR_IDMAP 0x00100000
+#  define MOUNT_ATTR_IDMAP 0x00100000
```

Comment:
Is the linter being silly? The space after `#` is not any style I'm familiar with... It smells as it thinks `#` is used for comments and it is indenting it as if it were comments